website/docs/docs/collaborate/explore-projects.md

```diff
@@ -12,15 +12,17 @@ With dbt Explorer, you can view your project's [resources](/docs/build/projects)
 
 - You have a dbt Cloud account on the [Team or Enterprise plan](https://www.getdbt.com/pricing/).
 - You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) or [staging](/docs/deploy/deploy-environments#create-a-staging-environment) deployment environment for each project you want to explore.
-    - There has been at least one successful job run in the deployment environment.
+    - There has been at least one successful job run in the deployment environment. Note, dbt Explorer doesn't get updated by [continuous integration (CI) jobs](/docs/deploy/ci-jobs).
 - You are on the dbt Explorer page. To do this, select **Explore** from the top navigation bar in dbt Cloud.
 
 
 ## Generate metadata 
 
 dbt Explorer uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). The metadata that's available depends on the [deployment environment](/docs/deploy/deploy-environments) you've designated as _production_ or _staging_ in your dbt Cloud project. dbt Explorer automatically retrieves the metadata updates after each job run in the production or staging deployment environment so it always has the latest results for your project. 
 
-To view a resource and its metadata, you must define the resource in your project and run a job in the production or staging environment. The resulting metadata depends on the [commands](/docs/deploy/job-commands) executed by the jobs.
+Note that CI jobs do not update dbt Explorer because they don't reflect the production state or provide the necessary metadata updates.
```

Comment:
```suggestion
Note that CI jobs do not update dbt Explorer. This is because they don't reflect the production state and don't provide the necessary metadata updates.
```